src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/Rules/Operations/IndentBlockOption.cs

```diff
@@ -37,6 +37,19 @@ internal enum IndentBlockOption
         /// <summary>
         /// Mask for position options
         /// </summary>
-        PositionMask = RelativeToFirstTokenOnBaseTokenLine | RelativePosition | AbsolutePosition
+        PositionMask = RelativeToFirstTokenOnBaseTokenLine | RelativePosition | AbsolutePosition,
```

Comment:
i think this should actually doc why this doesn't cover the item below.

Reply:
Indirectly documented. Let me know if it works.

Reply:
sorry, not seeing it, even after rereviewing things.

Reply:
`IndentIfConditionOfAnchorToken` is not a primary behavior.

Reply:
this should be documented.   the problem with the code/comemtns as they exist today is that i cannot come back and determine if this is an intentional or accidental omission.  i.e. the comment says "specifies one of the position options to indicate the primary behavior for the operation."  First, i don't know what "primary behavior" means.  Second, i have no way to tell if 'IndentIfConditionOfAnchorToken' is not primary, and thus is appropraite not in this list, or if it is primary, and this is just a mistake.

This is a prime example of why more is more when it comes to certain types of comments.  Just being explicit about intentions here goes a long way for future maintainability.